Web/Components/Pages/Login.razor.css

```
@@ -0,0 +1,74 @@
+::deep .landing-background {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)